### DISCUSSION TITLE
Innertune-like Notification Buttons

### DISCUSSION THREAD

**Discussion by HatoGreyrat** [libre-tube/LibreTube#6913] [Ideas]
### Describe your suggested feature

I would like to have buttons similar to Innertune in the notifications part to add to the library or favorites, but I think Libretube does not have a library or favorites, so you could add them or you could create a button to add directly from the notifications what you are listening to a selected playlist

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
I personally don't think that we should add more buttons to the notification, see https://github.com/libre-tube/LibreTube/issues/6868
